python/packages/isce3/atmosphere/split_band_estimation.py

```diff
@@ -322,11 +352,16 @@ def get_valid_area(
                     slant_main,
                     slant_side,
                     high_band_array)
-
-        mask_array = (high_band_array != invalid_value) & \
-                     (low_band_array != invalid_value) & \
-                     ~np.isnan(high_band_array) & \
-                     ~np.isnan(low_band_array)
+        if main_array is not None and diff_low_high_band_array is not None:
+            mask_array = (diff_low_high_band_array != invalid_value) & \
+                         (main_array != invalid_value) & \
+                         np.invert(np.isnan(diff_low_high_band_array)) & \
+                         np.invert(np.isnan(main_array))
```

Comment:
I don't think np.invert is needed here. Also you have used "~" in next block. So let's be consistent.
```suggestion
                         ~np.isnan(diff_low_high_band_array) & \
                         ~np.isnan(main_array)
```